test/liveValidatorTests.ts

```diff
@@ -605,6 +605,7 @@ describe("Live Validator", () => {
       const options = {
         directory: "./test/liveValidation/swaggers/specification/defaultIsErrorOnly",
         swaggerPathsPattern: ["test.json"],
+        enableRoundTripValidator: true
```

Comment:
why do you enable roundtrip for live validator tests? It's ok to have a new single test to validate roundtrip validation enablement doesn't impact live validation.